src/css/popup.css

```diff
@@ -1,545 +1,523 @@
+  background-color: var(--colorError);
+  color: var(--colorWhite);
 }
 
-.report-success h1 {
-  font-weight: 700;
-  font-size: 16px;
+.news-count.is-hidden {
+  display: none;
 }
 
-.report-continue {
-  cursor: pointer;
-  color: var(--relayBlue3);
+/* Default icon width */
+.fx-relay-menu-dashboard-link img {
+  pointer-events: none;
+  width: 14px;
+  filter: grayscale(1);
 }
 
-.setting {
-  padding: 12px 0;
-  border-bottom: 1px solid var(--relayGrey20);
-  display: flex;
-  justify-content: space-between;
+.fx-relay-menu-dashboard-link.is-active img,
+.fx-relay-menu-dashboard-link:hover img {
+  filter: grayscale(0);
 }
 
-.flex-col {
-  flex-direction: column;
+.fx-relay-menu-dashboard-link[data-panel-id="settings"] img {
+  width: 16px;
 }
 
-.setting-label {
-  display: inline-block;
-  color: var(--relayInk70);
+.fx-relay-menu-dashboard-link.is-active,
+.fx-relay-menu-dashboard-link:hover,
+.fx-relay-menu-dashboard-link:focus {
+  background-color: var(--colorGrey10);
+  border-radius: 100%;
 }
 
-a.setting-label {
-  text-decoration: none;
+.fx-relay-menu-dashboard-link.is-active .fx-relay-menu-dashboard-link-tooltip,
+.fx-relay-menu-dashboard-link .fx-relay-menu-dashboard-link-tooltip {
+  display: none;
+  color: var(--colorWhite);
+  background-color: var(--colorGrey40);
+  border-radius: var(--borderRadiusSm);
+  padding: var(--spacingXs) var(--spacingSm);
+  position: absolute;
+  top: calc(100% + var(--spacingXs));
+  right: 0;
+  white-space: nowrap;
 }
 
-a.setting-label:hover {
-  background-color: var(--relayGrey20);
+.fx-relay-menu-dashboard-link:hover .fx-relay-menu-dashboard-link-tooltip,
+.fx-relay-menu-dashboard-link:focus .fx-relay-menu-dashboard-link-tooltip,
+.fx-relay-menu-dashboard-link:focus-visible .fx-relay-menu-dashboard-link-tooltip {
+  display: block;
+  margin: 0 var(--spacingXs);
 }
 
-a.setting-label:focus {
-  box-shadow: var(--relayButtonFocus);
-}
+/* Panel Header */
 
-a.setting-label:active {
-  background-color: var(--relayGrey30);
+.fx-relay-panel-header {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
 }
 
-.settings-list {
-  padding: 0;
-  margin: 0;
-  list-style-type: none;
+.fx-relay-panel-header-btn-back {
+  position: absolute;
+  left: var(--spacingSm);
+  appearance: none;
+  background-color: transparent;
+  border: 0;
   margin: 0;
+  padding: var(--spacingXs);
+  cursor: pointer;
 }
 
-.setting-link {
-  margin-bottom: 12px;
+.fx-relay-panel-header-btn-back:focus,
+.fx-relay-panel-header-btn-back:hover {
+  background-color: var(--colorGrey10);
+  border-radius: 100%;
 }
 
-.setting-link:last-of-type {
-  margin-bottom: 0;
+.fx-relay-panel-header-btn-back img {
+  display: block;
+  width: 100%;
+  width: 24px;
+  height: 24px;
+  pointer-events: none;
 }
 
-fx-relay-logo-wrapper {
-  background-color: white;
+.fx-relay-panel-header-title {
+  font-size: var(--fontSizeBodyMd);
+  font-family: var(--relayMetropolis);
+  font-weight: 500;
```

Comment:
I try to avoid the font-short hand as a general rule,  as its more readable. 

That said, I do use short-hand for other stuff (`background`, `padding`, etc). We should determine as a team how we want to treat shorthand. 

cc/ @codemist @Vinnl

Reply:
I would lean towards what is more readable. No other pref. from my end. Thanks!